.github/workflows/event-pull_request.yml

```diff
@@ -37,12 +37,13 @@ jobs:
     secrets: inherit
 
   sanitize:
-    needs: docs-only
-    if: ${{ needs.docs-only.outputs.only-docs-changed == 'false' && !github.event.pull_request.draft }}
+    needs: [docs-only, get-latest-redis-tag]
+    if: ${{ needs.docs-only.outputs.only-docs-changed == 'false' && github.event.pull_request.draft }}
```

Comment:
why remove the "!"? now it will run only for drafts PRs (and in particular sanitizer did not run in this PR ...)

Reply:
the PR was on draft until now, so I needed it